src/proto/mapping_options.proto

```diff
@@ -43,24 +43,42 @@ message MappingOptionsProto {
   // Tile sizes.  If empty or not provided, do not tile.
   optional TilingProto tiling = 4;
   // Block dimensions (min 1, max 3 values).
-  required CudaDimProto block = 5;
+  // required CudaDimProto block = 5;
   // Grid dimensions (min 1, max 3 values).
-  required CudaDimProto grid = 6;
+  // required CudaDimProto grid = 6;
```

Comment:
Remove commented out "code".
Use `reserved` instead (if allowed by the version of protobuf we are using).